salt/api.sls

```diff
@@ -1,16 +1,47 @@
+#!jinja|yaml
+
 {% from "salt/map.jinja" import salt_settings with context %}
 
 include:
   - salt.master
+  - pip.extensions
+
+{%- set cfg_salt = pillar.get('salt', {}) %}
+{%- set cfg_master = cfg_salt.get('master', {}) %}
 
-salt-api:
-{% if salt_settings.install_packages %}
+salt_api_install:
   pkg.installed:
-    - name: {{ salt_settings.salt_api }}
-{% endif %}
+    - name: {{ salt_settings['salt_api'] }}
   service.running:
-    - name: {{ salt_settings.api_service }}
```

Comment:
This is correct. salt_settings.api_service can never be unset (it's in defaults.yaml).